lib/flood_risk_engine/services/address/deserialize/ea_facade_to_address.rb

```diff
@@ -0,0 +1,81 @@
+# Specific converter - convert EA AddressFacade into other usable forms
+
+module FloodRiskEngine
+  module Services
+    module Address
+      module Deserialize
+
+        class EaFacadeToAddress < Deserialize::DeserializeBase
+
+          def selectable_address_data(inbound)
+            addresses = inbound["results"] if inbound && inbound.is_a?(Hash)
+
+            addresses ||= []
+
+            results = addresses.collect do |a|
+              {
+                moniker: a["address"],
+                uprn: a["uprn"]
+              }
+            end
+
+            results
+          end
+
+          # Builds collection of attributes in correct format to simply
+          # pass to Address new or create - to build an Address (plus Location)
+
+          def address_data(inbound) # rubocop:disable Metrics/MethodLength:
+            addresses = inbound["results"] if inbound
+
+            addresses ||= []
+
+            results = addresses.collect do |a|
```

Comment:
I'm not sure this doing anything meaningful except setting organisation to `""` if absent. Addresses can be built with symbol or hash keys.